skills/csharp/experimental/bingsearchskill/Utilities/EngineWrapper.cs

```diff
@@ -57,12 +53,12 @@ public static Activity GetCardResponse(this LocaleTemplateEngineManager manager,
             }
         }
 
-        public static Activity GetCardResponse(this LocaleTemplateEngineManager manager, string templateId, Card card, StringDictionary tokens = null, string containerName = null, IEnumerable<Card> containerItems = null)
+        public static Activity GetCardResponse(this LocaleTemplateManager manager, string templateId, Card card, IDictionary<string, object> tokens = null, string containerName = null, IEnumerable<Card> containerItems = null)
```

Comment:
could you refer to the change in wrapper in ITSM skill? change GetResponse to GenerateActivity

Reply:
fixed